src/tokens/transformation/android/filters/lightColor.js

```diff
@@ -0,0 +1,5 @@
+const {
+  matchLightThemeToken
+} = require('../../common/tokenFilters')
+
+module.exports = (token) =>(matchLightThemeToken(token) || token.name.startsWith('color_primitive'))
```

Comment:
This doesn't seem quite right - all the primitive colors are included with the light colors?

Reply:
Could it be worth having three files:
1. Light
2. Dark
3. Primitive (which aren't themed)

Reply:
@fallaciousreasoning in android, we can have one .xml for light colors and one for dark colors. and android would use light .xml as default colors so i have put primitive colors inside light .xml as those colors aren't themed.